nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    riaise NotImplementedError
+  end
```

Comment:
- [ ] kindly check the correct name  method you have use riaise NotImplementedError  instead of raise NotImplementedError please correct this in other to avoid bugs